styles.css

```diff
@@ -56,35 +96,127 @@ ul {
 }
 
 .hero-wrapper {
-  margin-top: 10vh;
   height: 90vh;
+}
+
+.hero-container {
+  padding-bottom: 150px;
   background-image: url(/assets/svgs/header-shapes\ mobile.svg);
+  background-size: 100% 100%;
   background-repeat: no-repeat;
-  background-size: contain;
-  background-position: right;
 }
 
-.hero-container {
-  padding-top: 100px;
+.hero-container h1 {
+  padding-top: 140px;
 }
 
-.hero-wrapper h1 {
-  font-weight: 700;
-  font-size: 48px;
-  line-height: 52px;
+.social {
+  display: flex;
+  gap: 16px;
+}
+
+.projects-wrapper {
+  display: grid;
+}
+
+.project-container {
+  padding-top: 16px;
+  padding-bottom: 16px;
+}
+
+.project-snapshot {
+  width: 100%;
+  height: 42%;
+  display: block;
+}
+
+.project-snapshot img {
+  width: 100%;
+  height: 100%;
+}
+
+.project-container h2 {

```

Comment:
- [ ] Kindly add appropriate margins between the sections, and also between each project cards
Your designs
![Capture](https://user-images.githubusercontent.com/85095702/170688595-9a1eb1f3-0119-470a-9b20-c1c84515f550.jpg)
![Capture1](https://user-images.githubusercontent.com/85095702/170688636-49f3b655-f29e-49e6-98b5-2753c295e780.jpg)
![Capture2](https://user-images.githubusercontent.com/85095702/170688736-a2e1659b-31e1-4e47-9ff7-2af503eebcce.jpg)
Figma Designs
![figma](https://user-images.githubusercontent.com/85095702/170688793-be0b6ff2-8aed-4d6e-b442-05502b0018ce.jpg)
![figma2](https://user-images.githubusercontent.com/85095702/170688819-3795f892-bc16-49e3-a89e-a3b0502e6aaf.jpg)
![figma3](https://user-images.githubusercontent.com/85095702/170688833-dd6226b7-5b75-4cf3-b857-808d3929d1b7.jpg)